docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Amazon QuickSight
+description: Amazon QuickSight overview
+template: concept-topic-template
+last_updated: Oct 24, 2024
+---
+
+Amazon QuickSight brings customizable and powerful business intelligence tools to the Back Office.
+
+## Data sources and data sets
+
+This feature supports data from Spryker projects and third-party systems. When you set up Amazon QuickSight, the data from your project is available by default, and you can add other data sources if needed.
+
+![QuickSight-datasets](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-data-sets.png)
+
+For more details on data sources, see [Supported data sources](https://docs.aws.amazon.com/quicksight/latest/user/supported-data-sources.html).
+
+### Extending project data for analytics
+
+By looking at datasets, you can see what data from your project is available for adding to analytics. By default, for each dataset, only the data that can be useful for analytics is available from a respective database table. For example, the Customer dataset doesn't include customer passwords.
+
+You can extend the data available to datasets on the project level. For more details, see [Install Amazon QuickSight]().
+
+
+
+### Refreshing data
+
+To make datasets up to date, you need to refresh them by resyncing data from your project or third-party systems.
+
+You can refresh data manually or automatically by setting up a schedule per data set.
```

Comment:
What do you mean by `when you enable analytics in your project`? Can you set up automatic refreshing during the process of enabling analytics?

Reply:
It is about to activate the automatic update once you start working with Analytics data. Like you start using feature and one of the first things should be the scheduler set up so that the data is up to date. Doesn't have to be the FIRST thing you do but obviously very important to do.